lib/sisimai/bite/email/exim.rb

```diff
@@ -0,0 +1,522 @@
+                    v['lhost'] = cv[1]
+
+                  elsif cv = e.match(/\A[Aa]ction:[ ]*(.+)\z/)
+                    # Action: failed
+                    v['action'] = cv[1].downcase
+
+                  elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+                    # Status: 5.0.0
+                    v['status'] = cv[1]
+
+                  elsif cv = e.match(/\A[Dd]iagnostic-[Cc]ode:[ ]*(.+?);[ ]*(.+)\z/)
+                    # Diagnostic-Code: SMTP; 550 5.1.1 <userunknown@example.jp>... User Unknown
+                    v['spec'] = cv[1].upcase
+                    v['diagnosis'] = cv[2]
+
+                  elsif cv = e.match(/\A[Ff]inal-[Rr]ecipient:[ ]*(?:RFC|rfc)822;[ ]*(.+)\z/)
+                    # Final-Recipient: rfc822;|/bin/echo "Some pipe output"
+                    v['spec'] ||= cv[1] =~ /[@]/ ? 'SMTP' : 'X-UNIX'
+
+                  else
+                    # Error message ?
+                    if havepassed[:deliverystatus].zero?
+                      # Content-type: message/delivery-status
+                      havepassed[:deliverystatus] = 1 if e =~ Re1[:deliverystatus]
+                      v['alterrors'] ||= ''
+                      v['alterrors']  += e + ' ' if e =~ /\A[ ]+/
+                    end
+                  end
+                else
+                  if dscontents.size == recipients
+                    # Error message
+                    next unless e.size
+                    v['diagnosis'] ||= ''
+                    v['diagnosis']  += e + '  '
+
+                  else
+                    # Error message when email address above does not include '@'
+                    # and domain part.
+                    next unless e =~ /\A[ ]{4}/
+                    v['alterrors'] ||= ''
+                    v['alterrors']  += e + ' '
+                  end
+                end
+              end
+            end
+          end
+        end
+
+        if recipients > 0
+          # Check "an undisclosed address", "unroutable address"
+          dscontents.map do |q|
+            # Replace the recipient address with the value of "alias"
+            next unless q['alias']
+            next unless q['alias'].size > 0
+            if q['recipient'].empty? || q['recipient'] !~ /[@]/
+              # The value of "recipient" is empty or does not include "@"
+              q['recipient'] = q['alias']
+            end
+          end
+
+        else
+          # Fallback for getting recipient addresses
+          if mhead['x-failed-recipients']
+            # X-Failed-Recipients: kijitora@example.jp
+            rcptinhead = mhead['x-failed-recipients'].split(',')
+            rcptinhead.each do |a|
+              # Remove space characters
+              a.lstrip!
+              a.rstrip!
+            end
+            recipients = rcptinhead.size
+
+            rcptinhead.each do |e|
+              # Insert each recipient address into @$dscontents
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->